### PR TITLE
Add Context::connected

### PR DESCRIPTION
**Pull request by pandaman64**
If an actor registers an event, the actor does not stop when all addresses to it get dropped.
This behavior is not desirable in some cases like an actor periodically polls an expensive API and works as a cache.
This PR adds `connected` method to `Context`, which allows spawned future to check whether there is an alive address to the actor and to stop it when there is none.

### CODE REVIEW

**Comment by JohnTitor** [comment]
Could you also update CHANGES.md?

File: src/context.rs [hunk 1]
```diff
@@ -144,6 +144,10 @@ where
     pub fn set_mailbox_capacity(&mut self, cap: usize) {
         self.parts.set_mailbox_capacity(cap)
     }
+
+    pub fn connected(&self) -> bool {
```
Comment (vmalloc) [inline src/context.rs]: I would add a documentation comment here

**Comment by pandaman64** [comment]
updated.

**Comment by pandaman64** [comment]
Any chance get this merged?

**Comment by JohnTitor** [comment]
I don't have a strong opinion about this, any thoughts? @actix/contributors

**Pull request merged**